src/shared/src/backend-supports/projects-get-all.ts

```diff
@@ -12,7 +12,12 @@ const calculateEndDate = (start: Date, weeks: number) => {
   return end;
 };
 
-const projectDurationBuilder = (wps: any) => {
+interface WP {
```

Comment:
Can we rename this interface as **WorkPackageDurationProps** to give more details for the code readers? I'm unsure if Reid or Anthony will have a different opinion on naming this type.